packages/opentelemetry-context-zone-peer-dep/test/ZoneContextManager.test.ts

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import 'zone.js';
+import 'zone.js/dist/zone';
```

Comment:
I believe we should enable transpiling the zone.js with webpack if the default entry of zone.js contains advanced syntax.

This can be done in a follow-up PR.